app/assets/javascripts/modules/sortable_guiders.es6

```diff
@@ -6,11 +6,54 @@
     start(el) {
       super.start(el);
 
+      this.actionPanel = $('.js-action-panel');
+      this.saveButton = $('.js-save');
+
       this.init();
+      this.bindEvents();
     }
 
     init() {
-      new Sortable(this.$el[0]);
+      this.sortable = new Sortable(this.$el[0], {
+        onUpdate: () => this.hasListChanged()
+      });
+
+      this.pageOrder = this.getOrder();
+    }
+
+    bindEvents() {
+      this.saveButton.on('click', () => {
+        this.clearUnloadEvent();
+      });
+    }
+
+    getOrder() {
+      return JSON.stringify(this.sortable.toArray());
+    }
+
+    hasListChanged() {
```

Comment:
This is a bit of a weird method name for something that doesn't return anything.

How about just `onListUpdate` or something?